src/quacc/recipes/common/phonons.py

```diff
@@ -48,12 +52,22 @@ def phonon_subflow(
     """
     Calculate phonon properties.
 
+    In Quacc the ASE constraints can be used to fix atoms. These atoms will
+    not be displaced during the phonon calculation. This will greatly reduce
+    the computational cost of the calculation. However, this is an important
+    approximation and should be used with caution.
+
     Parameters
     ----------
     atoms
         Atoms object with calculator attached.
```

Comment:
While we're here, this docstring seems wrong: should just be "Atoms object" (it doesn't have a calculator attached)